HaxeManual/08-compiler-features.tex

```diff
@@ -169,7 +169,7 @@ \subsection{Overview}
 All completion is triggered using the \ic{--display file@position[@mode]} compiler argument. The required arguments are:
 
 \begin{description}
-	\item[file:] The file to check for completion. This must be an absolute or relative path to a .hx file. It does not respect any class paths or libraries.
+	\item[file:] The file to check for completion. This must be an absolute or relative path to a .hx file. It does not respect any classpaths or libraries.
```

Comment:
I might be wrong, but I think the majority of the manual uses `class paths`.

Reply:
I don't think so (https://en.wikipedia.org/wiki/Classpath_(Java)).

Reply:
yeah that's because I used `classpaths` instead of `class paths` @Gama11

Reply:
I don't follow.. do you mean "that's why"?

Reply:
@Gama11 yes